### PR TITLE
Ajax feedback

### PR DESCRIPTION
**Pull request by MaxShehovtsov**
Addressed feedback from here - https://github.com/Microsoft/ApplicationInsights-JS/issues/84

### CODE REVIEW

File: JavaScript/JavaScriptSDK/ajax/ajax.ts [hunk 1]
```diff
@@ -92,12 +88,25 @@ module Microsoft.ApplicationInsights {
             };
         }
 
+        private openHandler(xhr: XMLHttpRequestInstrumented, method, url, async) {
+            var ajaxData = new ajaxRecord();
+            ajaxData.method = method;
+            ajaxData.requestUrl = url;
+            ajaxData.requestSize += url.length;
+            // If not set async defaults to true 
+            ajaxData.async = extensions.IsNullOrUndefined(async) ? true : async;
```
Comment (SergeyKanzhelev) [inline JavaScript/JavaScriptSDK/ajax/ajax.ts]: As I mentioned we are making work to remove async attribute from schema. Also it's nullable - you should not default it to true.

**Comment by SergeyKanzhelev** [comment]
Looks good

File: JavaScript/JavaScriptSDK/ajax/ajax.ts (hunk 1 above)
Comment (MaxShehovtsov) [inline JavaScript/JavaScriptSDK/ajax/ajax.ts]: I'll remove async

**Comment by MaxShehovtsov** [comment]
@SergeyKanzhelev I removed async property from ajax (rdd) telemetry item

**Pull request merged**